cdp_scrapers/instances/portland.py

```diff
@@ -48,6 +52,28 @@
 
 ###############################################################################
 
+MATTER_ADOPTED_PATTERNS = [
+    "accepted",
+    "passed",
+    "adopted",
+    "confirmed",
+]
+MATTER_IN_PROG_PATTERNS = [
+    "passed to",
+    "placed on",
+    "continued",
+    "referred",
+]
+
+MINUTE_ITEM_PASSED_PATTERNS = [
+    "accepted",
+    "passed$",
```

Comment:
Is the `$` intentional?

Reply:
Yes it is. To differentiate e.g. "passed" and "passed to second reading".